composer.json

```diff
@@ -21,36 +21,37 @@
         }
     ],
     "require": {
-        "php": "^5.5.9 || ^7.0",
-        "paragonie/random_compat": "^1 || ^2 || ^9",
-        "symfony/form": "^2.8 || ^3.0 || ^4.0",
-        "symfony/framework-bundle": "^2.8 || ^3.0 || ^4.0",
-        "symfony/security-bundle": "^2.8 || ^3.0 || ^4.0",
-        "symfony/templating": "^2.8 || ^3.0 || ^4.0",
-        "symfony/translation": "^2.8 || ^3.0 || ^4.0",
-        "symfony/twig-bundle": "^2.8 || ^3.0 || ^4.0",
-        "symfony/validator": "^2.8 || ^3.0 || ^4.0",
-        "twig/twig": "^1.28 || ^2.0"
+        "php": "^7.2",
+        "doctrine/persistence": "^1.3",
+        "symfony/form": "^3.4 || ^4.4",
+        "symfony/framework-bundle": "^3.4 || ^4.4",
+        "symfony/security-bundle": "^3.4 || ^4.4",
+        "symfony/templating": "^3.4 || ^4.4",
+        "symfony/translation": "^3.4 || ^4.4",
+        "symfony/twig-bundle": "^3.4 || ^4.4",
+        "symfony/validator": "^3.4 || ^4.4",
+        "twig/twig": "^2.0 || ^3.0"
```

Comment:
can we already allow Twig 3.0 in the 2.x branch ? I think we are compatible.

Reply:
https://github.com/FriendsOfSymfony/FOSUserBundle/pull/2959